operational/flexera/custom_branding/README.md

```diff
@@ -0,0 +1,75 @@
+# Custom Branding

```

Comment:
Policy template name should be more clear about what it does. "Configure Custom Branding" or something like that would make it more clear.